src/core/tiledscene/qgstiledscenerendererregistry.cpp

```diff
@@ -17,13 +17,21 @@
 
 // default renderers
 #include "qgstiledscenetexturerenderer.h"
+#include "qgstiledscenetexturecolorrenderer.h"
+#include "qgstiledscenemeshrenderer.h"
 
 QgsTiledSceneRendererRegistry::QgsTiledSceneRendererRegistry()
 {
   // add default renderers
   addRenderer( new QgsTiledSceneRendererMetadata( QStringLiteral( "texture" ),
-               QObject::tr( "Textures" ),
+               QObject::tr( "Textured" ),
                QgsTiledSceneTextureRenderer::create ) );
+  addRenderer( new QgsTiledSceneRendererMetadata( QStringLiteral( "texturecolor" ),
+               QObject::tr( "Texture Color" ),
+               QgsTiledSceneTextureColorRenderer::create ) );
+  addRenderer( new QgsTiledSceneRendererMetadata( QStringLiteral( "mesh" ),
+               QObject::tr( "Mesh" ),
```

Comment:
Shall we cal this renderer "wireframe" ? It seems to me that term is more commonly used, and we would have fewer meaning associated with the "mesh" term...

Reply:
Good point :+1: 

I'm planning on adding rendering of line primitives there too, but a "wireframe" name still applies even with that!